<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editsequence</class>
 <widget class="QDialog" name="editsequence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1884</width>
    <height>1133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Sequence</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0">
   <item row="0" column="0">
    <widget class="QLabel" name="lb_edit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Edit Test Sequence</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cb_seq">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Sequence1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sequence2</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="pb_insertrow">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 255);</string>
      </property>
      <property name="text">
       <string>Insert Row</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_delrow">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Delete Row</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_saveseq">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 127);</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTableWidget" name="tableseq">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
